src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/GrunnlagsdataRegisterService.kt

```diff
@@ -33,7 +33,7 @@ class GrunnlagsdataRegisterService(
         val medlUnntak = personopplysningerIntegrasjonerClient.hentMedlemskapsinfo(ident = personIdent)
 
         val tidligereVedtaksperioder =
-            tidligereVedaksperioderService.hentTidligereVedtaksperioder(pdlSøker.alleIdenter())
+            tidligereVedaksperioderService.hentTidligereVedtaksperioder(pdlSøker.folkeregisteridentifikator)
```

Comment:
Nå har vi ikke åpnet for npid, men kan ikke denne bli litt vond hvis vi skal/må gjøre det?